plugins/obs-webrtc/whip-output.cpp

```diff
@@ -368,6 +424,24 @@ bool WHIPOutput::Connect()
 	CURLU *url_builder = curl_url();
 	auto last_location_header = location_headers.back();
 
+	struct curl_header *prev = nullptr;
+	struct curl_header *h = nullptr;
+	std::vector<rtc::IceServer> iceServers;
+	while ((h = curl_easy_nextheader(c, CURLH_HEADER, 0, prev))) {
+		if (strcmp("Link", h->name) == 0 ||
+		    strcmp("link", h->name) == 0) {
```

Comment:
Are these the only two variations of "link" that are valid? If any case variation is valid, consider `astrcmpi` instead. If these are the only two valid forms, then this is fine.

Reply:
Switched to `astrcmpi`. It makes the code simpler and more forgiving incase some does something like `LINK` and doesn't realize it.